crates/graphql/src/schema/dataloaders/stats.rs

```diff
@@ -38,3 +39,19 @@ impl TryBatchFn<PublicKey<StoreConfig>, Option<MarketStats>> for Batcher {
             .batch(addresses))
     }
 }
+
+#[async_trait]
+impl TryBatchFn<PublicKey<StoreCreator>, Option<StoreCreatorStats>> for Batcher {
+    async fn load(
+        &mut self,
+        addresses: &[PublicKey<StoreCreator>],
+    ) -> TryBatchMap<PublicKey<StoreCreator>, Option<StoreCreatorStats>> {
+        let db = self.db()?;
+        let rows = stats::store_creator(&db, addresses)?;
+
+        Ok(rows
+            .into_iter()
+            .map(|s| (s.store_creator.clone(), s.try_into()))
+            .batch(addresses))
+    }
+}
```

Comment:
Lets move the data loader to store_creators.
```suggestion
#[async_trait]
impl TryBatchFn<PublicKey<StoreCreator>, Option<StoreCreatorStats>> for Batcher {
    async fn load(
        &mut self,
        addresses: &[PublicKey<StoreCreator>],
    ) -> TryBatchMap<PublicKey<StoreCreator>, Option<StoreCreatorStats>> {
        let db = self.db()?;
        let rows = stats::store_creator(&db, addresses)?;

        Ok(rows
            .into_iter()
            .map(|s| (s.store_creator.clone(), s.try_into()))
            .batch(addresses))
    }
}
```
```suggestion
#[async_trait]
impl TryBatchFn<PublicKey<StoreCreator>, Option<StoreCreatorStats>> for Batcher {
    async fn load(
        &mut self,
        addresses: &[PublicKey<StoreCreator>],
    ) -> TryBatchMap<PublicKey<StoreCreator>, Option<i64>> {
        let db = self.db()?;
        let rows = nft_counts::store_creators(&db, addresses)?;

        Ok(rows
            .into_iter()
            .map(|s| (s.store_creator.clone(), s.try_into()))
            .batch(addresses))
    }
}
```